servers/rendering/renderer_rd/shaders/environment/sky.glsl

```diff
@@ -83,20 +83,23 @@ layout(set = 0, binding = 1, std430) restrict readonly buffer GlobalShaderUnifor
 global_shader_uniforms;
 
 layout(set = 0, binding = 2, std140) uniform SceneData {
-	bool volumetric_fog_enabled;
-	float volumetric_fog_inv_length;
-	float volumetric_fog_detail_spread;
-
-	float fog_aerial_perspective;
-
-	vec3 fog_light_color;
-	float fog_sun_scatter;
-
-	bool fog_enabled;
-	float fog_density;
-
-	float z_far;
-	uint directional_light_count;
+	bool volumetric_fog_enabled; // 4 - 4
+	float volumetric_fog_inv_length; // 4 - 8
+	float volumetric_fog_detail_spread; // 4 - 12
+	float volumetric_fog_sky_affect; // 4 - 16
+
+	bool fog_enabled; // 4 - 20
+	float fog_sky_affect; // 4 - 24
+	float fog_density; // 4 - 28
+	float fog_sun_scatter; // 4 - 32
+
+	vec3 fog_light_color; // 12 - 44
+	float fog_aerial_perspective; // 4 - 48
+
+	float z_far; // 4 - 52
+	uint directional_light_count; // 4 - 56
+	// FIXME: This must be commented for the shader to work, why? See renderer_scene_sky_rd.h:162
+	//uint pad[2]; // 8 - 64
```

Comment:
This probably just needs to be a two variables instead of one. Std140 layout rules dictate that array strides will be padded to 16 bytes.